orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteTaskHandlerTest.kt

```diff
@@ -50,137 +50,83 @@ object CompleteTaskHandlerTest : SubjectSpek<CompleteTaskHandler>({
 
   fun resetMocks() = reset(queue, repository, publisher)
 
-  describe("when a task completes successfully") {
-    given("the stage contains further tasks") {
-      val pipeline = pipeline {
-        stage {
-          type = multiTaskStage.type
-          multiTaskStage.buildTasks(this)
-        }
-      }
-      val message = CompleteTask(
-        pipeline.type,
-        pipeline.id,
-        pipeline.application,
-        pipeline.stages.first().id,
-        "1",
-        SUCCEEDED
-      )
-
-      beforeGroup {
-        whenever(repository.retrieve(PIPELINE, message.executionId)) doReturn pipeline
-      }
-
-      afterGroup(::resetMocks)
-
-      action("the handler receives a message") {
-        subject.handle(message)
-      }
-
-      it("updates the task state in the stage") {
-        verify(repository).storeStage(check {
-          it.tasks.first().apply {
-            assertThat(status).isEqualTo(SUCCEEDED)
-            assertThat(endTime).isEqualTo(clock.millis())
+  setOf(SUCCEEDED, FAILED_CONTINUE).forEach { successfulStatus ->
```

Comment:
TBH this is the only change -- everything else is fallout from indenting this block

Reply:
Wish GitHub could ignore whitespace changes like IntelliJ can.